benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -825,14 +839,16 @@ void GraphicsBenchmarkApp::Render()
     frame.sceneData.viewProjectionMatrix = mCamera.GetViewProjectionMatrix();
 #if defined(PPX_BUILD_XR)
     if (IsXrEnabled()) {
-        const glm::mat4 v                    = GetXrComponent().GetViewMatrixForCurrentView();
-        const glm::mat4 p                    = GetXrComponent().GetProjectionMatrixForCurrentViewAndSetFrustumPlanes(PPX_CAMERA_DEFAULT_NEAR_CLIP, PPX_CAMERA_DEFAULT_FAR_CLIP);
+        const glm::mat4 v                    = GetXrComponent().GetViewMatrixForView(mViewIndex);
+        const glm::mat4 p                    = GetXrComponent().GetProjectionMatrixForViewAndSetFrustumPlanes(mViewIndex, PPX_CAMERA_DEFAULT_NEAR_CLIP, PPX_CAMERA_DEFAULT_FAR_CLIP);
         frame.sceneData.viewProjectionMatrix = p * v;
     }
 #endif
 
     RecordCommandBuffer(frame, swapchain, imageIndex);
 
+    swapchain->Wait(imageIndex);
```

Comment:
Not sure if adding that brings real-life benefits. Does moving this into the acquire to have something closer to the non-XR variant affects performance?

Not a strong opinion, feel free to dismiss this.